xgi/drawing/draw.py

```diff
@@ -1664,160 +1708,465 @@ def draw_dihypergraph(
+        pos[i + n] = edge_pos[e]
+
+    # compute axis limits
+    _update_lims(pos, ax)
+
+    ax.set_aspect(aspect, "datalim")
+
+    return ax, (node_collection, edge_marker_collection, dyad_collection)
+
+
+def draw_undirected_dyads(
+    H,
+    pos=None,
+    ax=None,
+    max_order=None,
+    dyad_color="black",
+    dyad_lw=1,
+    dyad_style="solid",
+    dyad_color_cmap="Greys",
+    rescale_sizes=True,
+    **kwargs,
+):
+    """Draw the bipartite edges of an undirected hypergraph.
+
+    Parameters
+    ----------
+    H : Hypergraph
+        The hypergraph to draw.
+    pos : tuple of two dicts, optional
+        The tuple should contains a (1) dictionary of positions node_id:(x,y) for
+        placing node markers, and (2) a dictionary of positions edge_id:(x,y) for
+        placing the edge markers.  If None (default), use the `bipartite_spring_layout`
+        to compute the positions.
+    ax : matplotlib.pyplot.axes, optional
+        Axis to draw on. If None (default), get the current axes.
+    max_order : int, optional
+        Maximum of hyperedges to plot. If None (default), plots all orders.
+    dyad_color : str, dict, iterable, optional
+        Color of the bipartite edges. If str, use the same color for all edges.
+        If a dict, must contain (hyperedge_id: color_str) pairs. If other iterable,
+        assume the colors are specified in the same order as the hyperedges are found
+        in DH.edges. By default, "black".
+    dyad_lw : int, float, dict, iterable, optional
+        Line width of the bipartite edges. If int or float, use the same width for
+        all hyperedges. If a dict, must contain (hyperedge_id: width) pairs. If other
+        iterable, assume the widths are specified in the same order as the hyperedges
+        are found in DH.edges. By default, 1.
+    dyad_style : str or list of strings, optional
+        Line style of the dyads, e.g. ‘-’, ‘–’, ‘-.’, ‘:’ or words like ‘solid’ or ‘dashed’.
+        See matplotlib's documentation for all accepted values. By default, "solid".
+    dyad_color_cmap : colormap
+        Colormap for mapping bipartite edge colors. By default, "Greys".
+        Ignored, if `dyad_color` is a str (single color) or an iterable of colors.
+    rescale_sizes: bool, optional
+        If True, linearly interpolate `node_size` and between min/max values
+        that can be changed in the other argument `params`.
+        If `node_size` is a single value, this is ignored. By default, True.
+    **kwargs : optional args
+        Alternate default values. Values that can be overwritten are the following:
+        * min_dyad_lw (default: 1)
+        * max_dyad_lw (default: 10)
+
+    Returns
+    -------
+    ax : matplotlib.pyplot.axes
+        The axes corresponding the visualization
+    * dyad_collection : matplotlib LineCollection
+        of bipartite edges
+
+    Raises
+    ------
+    XGIError
+        If something different than a DiHypergraph is passed.
```

Comment:
```suggestion
        If something different than a Hypergraph is passed.
```

Reply:
But I don't see that it raises an error, does it?

Reply:
Good catch! Fixed.